Error build on Xcode `make project`

# Reproduce Method
1. Clone git.
2. Run command `make project`
3. Xcode opened, then run `cmd + B` or build.
4. Build failed.

```
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "SwiftAndObjc.h"
        ^
/<your env dir>/_p/1YWHeOsVjJ-pub/SwiftAndObjc/SwiftAndObjc.h:2:9: error: 'SwiftAndObjc/SwiftAndObjc-Swift.h' file not found
#import <SwiftAndObjc/SwiftAndObjc-Swift.h>
        ^
<unknown>:0: error: could not build Objective-C module 'SwiftAndObjc'
```

# Xcode Version
`10.1`

--- a/README.md
+++ b/README.md
@@ -7,8 +7,7 @@ This is a sample to show how to use Buck to build an iOS project with mixed lang
 
 ```sh
 # Install Buck
-brew tap facebook/fb
-brew install buck
+make install_buck
 ```
 
 ### Running Tests

